tests/unit/babylon/src/Loading/babylon.sceneLoader.tests.ts

```diff
@@ -731,4 +731,94 @@ describe('Babylon Scene Loader', function() {
             return BABYLON.SceneLoader.LoadAsync("file:///", "Box.gltf").then(resetPreprocessUrl, resetPreprocessUrl);
         });
     });
-});
+
+    describe('#DirectLoad', () => {
+        it('should load a base64 encoded obj with no mime type', () => {
+            var scene = new BABYLON.Scene(subject);
+            return BABYLON.SceneLoader.LoadAssetContainerAsync('data:;base64,' + objBase64, undefined, scene, () => { }, ".obj").then((container) => {

```

Comment:
Is there a reason why sometimes we use asset container and sometimes not in these tests?

Reply:
That was just because I copy and pasted lines from different locations. I've updated them all to consistently use `ImportMeshAsync` now.